docs/execution-analysis/simulation.md

```diff
@@ -24,18 +26,18 @@ print(actual.tolist())
 print(simulation.tolist())
 ```
 
-After the simulation runs, it prints the following:
+After the simulation runs, it prints the following results:
 
 ```
 [1, 4, 9, 16, 25, 36, 49, 64, 81, 100]
 [1, 4, 9, 16, 16, 36, 49, 64, 81, 100]
 ```
 
 {% hint style="warning" %}
-There are some operations which are not supported in simulation yet. They will result in compilation failures. You can revert to simulation using graph execution using `circuit.graph(...)` instead of `circuit.simulate(...)`, which won't simulate FHE, but it will evaluate the computation graph, which is like simulating the operations without any errors due to FHE.
+Some operations are not yet supported in simulation, resulting in compilation failures. In such cases, you can revert to graph execution using `circuit.graph(...)` instead of `circuit.simulate(...)`. While this method doesn't simulate FHE, it evaluates the computation graph, allowing you to simulate the operations without FHE-related errors.
```

Comment:
Question to @umut-sahin (not blocking this PR): "Some operations are not yet supported in simulation, resulting in compilation failures.". Really, which ones?

Reply:
Indeed I don't see what cannot be simulated.. We should simulate everything as we have all crypto operators simulated.

Reply:
CRT was not simulated before. I'll remove this.